src/components/pages/secret-page.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+
+const SecretPage = ({isLoggedIn}) => {
+  if (isLoggedIn) {
```

Comment:
it's okay, but you can use something like that
`const ok = (<div>...</div>);
const redirect = (<Redirect to='' />);
return isLoggedIn ? ok : redirect;
`

Reply:
Keep things clean and simple, right. Nice!

Reply:
Done